_topic_map.yml

```diff
@@ -2127,6 +2127,8 @@ Name: Backup and restore
 Dir: backup_and_restore
 Distros: openshift-origin,openshift-enterprise
 Topics:
+- Name: Overview of nodes
```

Comment:
Please confirm that the title here is what you expect.

Reply:
The title looks wrong; have changed it now.